Library/Koala/Koala.swift

```diff
@@ -2043,10 +2048,16 @@ public final class Koala {
   private func defaultProperties() -> [String: Any] {
     var props: [String: Any] = [:]
 
+    let enabledFeatureFlags = self.config?.features
+      .filter { key, value in key.starts(with: "ios_") && value }
+      .keys
+      .sorted()
+
     props["manufacturer"] = "Apple"
     props["app_version"] = self.bundle.infoDictionary?["CFBundleVersion"]
     props["app_release"] = self.bundle.infoDictionary?["CFBundleShortVersionString"]
-    props["current_variants"] = AppEnvironment.current.config?.abExperimentsArray
+    props["current_variants"] = self.config?.abExperimentsArray.sorted() ?? []
```

Comment:
And here...

Reply:
I originally just had it as an optional but thought it might be nicer on the consuming side to see the property as empty rather than not see it at all 🤔

Reply:
Hm I see what you mean... For other optional properties we don't send anything, though. But this is definitely not a blocker. :)